libraries/chain/database.cpp

```diff
@@ -5476,10 +5476,7 @@ void database::validate_invariants()const
 
       for ( auto itr = smt_ico_idx.begin(); itr != smt_ico_idx.end(); ++itr )
       {
-         if ( get< smt_token_object, by_symbol >( itr->symbol ).phase <= smt_phase::launch_failed )
-            total_supply += asset( itr->contributed.amount - itr->processed_contributions, STEEM_SYMBOL );
-         else
-            total_supply += asset( itr->contributed.amount, STEEM_SYMBOL );
+         total_supply += asset( itr->contributed.amount - itr->processed_contributions, STEEM_SYMBOL );
```

Comment:
This was probably removed by mistake. I think this change causes `smt_ico_payouts` and `smt_ico_payouts_special_destinations` to fail.